assets/app/views/modals/process-template.html

```diff
@@ -0,0 +1,21 @@
+<div class="modal-resource-process">
+  <div class="modal-body">
+    <h1>You are adding a Template</h1>
+    <p>Please select type of actions you want to do with your defined Template:</p>
+
+    <div>
+      <input type="checkbox" ng-model="templateOptions.process"/>
+      <strong>Process the template</strong>
+      <span id="helpBlock" class="help-block">This will create an instance of the things defined in the template, you will have an opportunity to fill out any parameters the template defines.</span>
+    </div>
+    <div>
+    <input type="checkbox" ng-model="templateOptions.add"/>
+    <strong>Add template</strong>
+    <span id="helpBlock" class="help-block">This will save the tamplate and it available to others who have access to this project.</span>
+    </div>
+  </div>
```

Comment:
We should look at the wording in this dialog. Maybe:

**Add Template**

Select the actions to perform.

[x] Process template
Creates the objects defined in the template. You will have an opportunity to fill in template parameters.

[x] Save template
Saves the template to the project.

@jwforres suggestions?